expression/expression.go

```diff
@@ -165,9 +165,7 @@ func IsEQCondFromIn(expr Expression) bool {
 	if !ok || sf.FuncName.L != ast.EQ {
 		return false
 	}
-	cols := make([]*Column, 0, 1)
-	cols = ExtractColumnsFromExpressions(cols, sf.GetArgs(), isColumnInOperand)
```

Comment:
We can remove the function `IsColumnInOperand` now.

Reply:
removed